src/components/MoreOrLess/web/MoreOrLess.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
How about renaming this component to `Article` ?
I think `Article` makes much more sense here.

Reply:
I mean articles are place we might want to use this kind of components as they lengthy.